### ISSUE TITLE
Who were/are your mentors?

### ISSUE THREAD

**Issue by arnellebalane** [passy/ama#33]
And how did you get them to mentor you? :)

**Comment by passy** [comment]
Lovely question.

There are a lot of people I think of as mentors in a particular stage of my life and career. When it comes to open source, it's @sindresorhus and @addyosmani.

This all started with me participating in 24 Pull Requests in 2012 and contributing to a pretty random selection of mostly Python and JavaScript projects. I contributed a few small consistency patches to TodoMVC and got by far the nicest (and fastest) response, so I kept the contributions coming. Fast-forward a month or so, 24 PRs was long over and I was still submitting patches. Addy sent me an email asking if I wanted to join their team as third team member. It may sound cheesy, but I still remember that day as if it was yesterday; it was definitely one of the most exciting days in my life - and in hindsight an absolute game changer.

Addy and Sindre have taught me a lot about maintaining large open source projects, how painful and necessary rejecting other people's work is, how much time it takes to keep something running that looks incredibly simple from the outside and that staying motivated in the long-term requires work, too.

Luckily, in the open source community there are tons of people out there willing to help and teach other people. This is why I still <3 it so much.
